action.yml

```diff
@@ -20,6 +20,10 @@ inputs:
     required: false
     default: 'all'
     description: 'The type of tests to be run by the test runner.'
+  coverageOptions:
+    required: false
+    default: 'enableCyclomaticComplexity;generateHtmlReport;generateBadgeReport'
```

Comment:
The `enableCyclomaticComplexity` option has been replaced with the `generateAdditionalMetrics` option since version 0.3.0 of the Code Coverage package

Source: https://docs.unity3d.com/Packages/com.unity.testtools.codecoverage@1.2/changelog/CHANGELOG.html

Reply:
Updated in #183, release in [`v2.0.2`](https://github.com/game-ci/unity-test-runner/releases/tag/v2.0.2)